static/src/stylesheets/amp/_helpers.scss

```diff
@@ -78,3 +78,9 @@
     margin: 0;
     padding: 0;
 }
+
+.mobile-only {
```

Comment:
Amp is already mobile-only so surely this isn't needed?

Reply:
The caption show/hide functionality ends at tablet breakpoint (in the shared file). So that's why AMP was looking broken when the browser is above tablet breakpoint (Like in James' screenshot). 

Illustrates some potential issues with shared files between amp and the main site. 

Realise this isn't a likely thing for users to see, but I guess they could on a larger tablet maybe?

Reply:
Also, AMP validation is failing because `!important` isn't allowed

Reply:
Maybe we shouldn't share this file then :woman_shrugging: